Scripts/Python/plasma/Plasma.py

```diff
@@ -827,6 +827,10 @@ def PtStartScreenCapture(selfKey,width=800,height=600):
     """Starts a capture of the screen"""
     pass
 
+def PtSupportsPlanarReflections() -> bool:
+    """Returns if planar reflections are supported"""
+    ...
```

Comment:
For consistency with the rest of the file, and because this isn't a .pyi file:

```suggestion
    pass
```

Reply:
This is the way that I've been doing all the rest of the new/updated API functions to ease the (eventual? hopeful?) transition to pyi. Example:

https://github.com/H-uru/Plasma/blob/5807230e6e43e1f80e5b24c4878b468330710f2a/Scripts/Python/plasma/Plasma.py#L879-L883

Reply:
Ah, alright then - I guess I never noticed any of those yet. Yes, it would be nice to convert these to proper .pyi files at some point.